spec/lib/topic_query_spec.rb

```diff
@@ -1379,6 +1379,11 @@ def clear_cache!
       it "should return the new topic" do
         expect(TopicQuery.new.list_suggested_for(topic).topics).to eq([new_topic])
       end
+
+      it "should return the nothing when random topics excluded" do
+        # this API was added so plugins can generate a suggested topics list without the random topics
```

Comment:
I feel like this comment should be in `TopicQuery#list_suggested_for`, not in the spec

Reply:
sure will move it!